lib/config/metering/src/index.js

```diff
@@ -121,7 +120,7 @@ const id = (oid, rtype, ppid, time, auth, cb) => {
   if (!rtype) {
     debug('Undefined resource type');
     return cb(undefined,
-      businessResponse(true, 'Metering config - Undefined resource type'));
+      businessError('Metering config - Undefined resource type'));
```

Comment:
This could also be named simply ```error```, as that'd also be more consistent with the ```error``` field it returns.